src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,17 @@
+//colors used in card, header:
+$white-color-usein-header-card: #fff;
+//colors used in card, nav:
+$blue-color-usein-nav-card: #00acdc;
+//colors use in card:
+$price-value-color: #060b35;
+//colors use in card:
```

Comment:
don't add comments with variables usage
We can use these variables in two or more places